src/phoenix/session/session.py

```diff
@@ -133,6 +140,28 @@ def url(self) -> str:
         """Returns the url for the phoenix app"""
         return _get_url(self.port, self.is_colab)
 
+    def get_spans(
+        self,
+        filter_condition: Optional[str] = None,
+        *,
+        start_time: Optional[datetime] = None,
+        stop_time: Optional[datetime] = None,
+        root_spans_only: Optional[bool] = None,
+    ) -> Optional[pd.DataFrame]:
```

Comment:
Since we have a `Span` type, we might want to reserve `get_spans`. What do you think of `get_span_dataframe`?